plonk-core/src/constraint_system/arithmetic.rs

```diff
@@ -460,8 +456,35 @@ mod test {
             test_incorrect_big_arith_gate
         ],
         [] => (
-            Bls12_377,
-            ark_ed_on_bls12_377::EdwardsParameters
+            Bls12_377, ark_ed_on_bls12_377::EdwardsParameters
+        )
+    );
+    batch_test_ipa!(
+        [
+            test_public_inputs,
+            test_correct_add_mul_gate,
+            test_correct_add_gate,
+            test_correct_big_add_mul_gate,
+            test_correct_big_arith_gate,
+            test_incorrect_add_mul_gate,
+            test_incorrect_big_arith_gate
+        ],
+        [] => (
+            Bls12_381, ark_ed_on_bls12_381::EdwardsParameters
         )
     );
+
+    #[test]
+    #[allow(non_snake_case)]
+    fn test_ipa() {
```

Comment:
Is this test necessary?

Reply:
leftover from when I was just testing whether swapping IPA for KZG10 worked at all, so I'll remove it